app/gui/src/project-view/components/visualizations/TableVisualization.vue

```diff
@@ -745,8 +745,8 @@ config.setToolbar(
   overflow: hidden;
 }
 
-.TableVisualization > .ag-theme-alpine > :deep(.ag-root-wrapper.ag-layout-normal) {
-  border-radius: 0 0 var(--radius-default) var(--radius-default);
+.TableVisualization:deep(.ag-root-wrapper) {
```

Comment:
Does this still need a `:deep` selector?

Reply:
Yes, the `.ag-root-wrapper` is generated outside of vue's context, so it doesn't logically belong to a component. That makes `:deep` necessary.

Reply:
Of course `:deep` is necessary if `.ag-root-wrapper` is needed. Is `.ag-root-wrapper` needed?

Reply:
It is, because otherwise the variable is overriden at the ag-grid root element level.